include/bindey/property.h

```diff
@@ -7,10 +7,24 @@
 namespace bindey
 {
 
-template <typename T, typename UpdatePolicy = std::not_equal_to<T>>
+/**
+ * Optional AlwaysUpdate policy to notify subscribers everytime the property value is set, not just when it changes
+ */
+class AlwaysUpdate
```

Comment:
actually let's change this typename to be lower case `always_update` to match the rest of the types